io.github.theforceengine.tfe.appdata.xml

```diff
@@ -0,0 +1,53 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">io.github.theforceengine.tfe</id>
+  <name>The Force Engine</name>
+  <developer_name>The Force Engine Contributors</developer_name>
+  <summary>A modern cross-platform port of Star Wars: Dark Forces with mod support</summary>
```

Comment:
```suggestion
  <summary>Modern cross-platform port of Star Wars: Dark Forces with mod support</summary>
```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#dont-start-with-an-article